vote.py

```diff
@@ -133,13 +133,13 @@ def askyesno(self, event):
 
     def question_result(self, event=None):
         """Return results of question"""
-        res = ''
+        res = []
         for k, v in self.answers.iteritems():
-            res += '{0} -> {1}, '.format(v[0], v[1])
+            res.append('{0} -> {1}'.format(v[0], v[1]))
```

Comment:
@Adman meh, this could really be cleaner if you could do it via [map](http://www.bogotobogo.com/python/python_fncs_map_filter_reduce.php) or list comprehensions:

```python
res = ['{0} -> {1}'.format(v[0], v[1])) for _, v in self.answers.iteritems()]
```

This is just outright unpythonic =)